lighthouse-core/report/html/renderer/element-screenshot-renderer.js

```diff
@@ -206,13 +206,13 @@ class ElementScreenshotRenderer {
    * Returns null if element rect is outside screenshot bounds.
    * @param {DOM} dom
    * @param {ParentNode} templateContext
-   * @param {LH.Audit.Details.FullPageScreenshot} fullPageScreenshot
+   * @param {LH.Artifacts.FullPageScreenshot['screenshot']} screenshot
    * @param {LH.Artifacts.Rect} elementRectSC Region of screenshot to highlight.
    * @param {Size} maxRenderSizeDC e.g. maxThumbnailSize or maxLightboxSize.
    * @return {Element|null}
    */
-  static render(dom, templateContext, fullPageScreenshot, elementRectSC, maxRenderSizeDC) {
-    if (!screenshotContainsRect(fullPageScreenshot, elementRectSC)) {
+  static render(dom, templateContext, screenshot, elementRectSC, maxRenderSizeDC) {
+    if (!screenshotOverlapsRect(screenshot, elementRectSC)) {
```

Comment:
why this change? did we already support this and were being careful the first time around?

Reply:
Specifically, I was testing using some test fixture and saw that an element I knew was in the page bounds was not showing an element screenshot. I forget which.

Brendan did bring it up before https://github.com/GoogleChrome/lighthouse/pull/11538#discussion_r501914470 but clearly it's more common than I assumed since a random test case surfaced it, so I just fixed it here.